components/director/internal/domain/formation/service.go

```diff
@@ -1763,3 +1770,44 @@ func isObjectTypeSupported(formationTemplate *model.FormationTemplate, objectTyp
 
 	return true
 }
+
+// updateFormationAssignmentsWithStateForObjectID lists all formation assignments for a given objectID that are either source or target of the assignment
+// and update every one of them with the provided state
+func (s *service) updateFormationAssignmentsWithStateForObjectID(ctx context.Context, formationID, objectID, objectType, faState string) error {
```

Comment:
Can we rename the last argument to newState so that it is explicit what the argument is used for